frontend/test/metabase/scenarios/filters/view.cy.spec.js

```diff
@@ -21,7 +21,7 @@ describe("scenarios > question > view", () => {
       openOrdersTable();
```

Comment:
the Vendor column are now cached in AppDB, so we will have `Search the list` placeholder instead.